src/main/java/com/fuem/repositories/NotificationDAO.java

```diff
@@ -53,4 +71,94 @@ public List<Notification> getNotificationsForUser(int userId) {
 
         return notifications;
     }
+
+    /**
+     *
+     * @author AnhNQ Insert and return the id of new notification
+     */
```

Comment:
- author với code explain tách làm 2 dòng
- Chỉ nên giải thích logic, cách dùng phức tạp trách nói điều hiển nhiên
```suggestion
    /**
     * @return id of notification have been inserted
     * @author AnhNQ
     */
```